experiments/AtmosLES/cfsite_hadgem2-a_07_amip.jl

```diff
@@ -0,0 +1,631 @@
+# General Julia modules
+using ArgParse
+using Dierckx
+using Distributions
+using DocStringExtensions
+using LinearAlgebra
+using NCDatasets
+using Pkg.Artifacts
+using Printf
+using Random
+using StaticArrays
+using Test
+
+# ClimateMachine Modules
+using ClimateMachine
+
+using ClimateMachine.ArtifactWrappers
+using ClimateMachine.Atmos
+using ClimateMachine.ConfigTypes
+using ClimateMachine.GenericCallbacks
+using ClimateMachine.DGMethods.NumericalFluxes
+using ClimateMachine.Diagnostics
+using ClimateMachine.Mesh.Filters
+using ClimateMachine.Mesh.Grids
+using ClimateMachine.Orientations
+using ClimateMachine.Thermodynamics
+using ClimateMachine.TurbulenceClosures
+using ClimateMachine.ODESolvers
+using ClimateMachine.VariableTemplates
+using ClimateMachine.Writers
+
+# Planet parameters 
+using CLIMAParameters
+using CLIMAParameters.Planet: e_int_v0, grav, day
+struct EarthParameterSet <: AbstractEarthParameterSet end
+const param_set = EarthParameterSet()
+# Physics specific imports 
+using ClimateMachine.Atmos: altitude, recover_thermo_state
+import ClimateMachine.Atmos: source!, atmos_source!
+
+# Citation for problem setup
+## CMIP6 Test Dataset - cfsites
+## [Webb2017](@cite)
+
+struct GCMRelaxation{FT} <: AbstractSource
+    τ_relax::FT
+end
+function atmos_source!(
```

Comment:
Should we use the new tendency specification? If so, I can help by adding a commit or two

Reply:
Yes, please feel free to add commits! Thanks!